platforms/qurt/src/px4_layer/qurt_stubs.c

```diff
@@ -118,3 +118,42 @@ void __cxa_pure_virtual()
 	PX4_WARN("Error: Calling unresolved symbol stub[%s]", __FUNCTION__);
 	block_indefinite();
 }
+
+#ifdef __PX4_QURT_EAGLE_APQ8074
```

Comment:
So here I think `__PX4_QURT_EAGLE` is enough.

Reply:
Why is it needed at all? This is the only target building `qurt_stubs.c`.

Reply:
I suppose your right.

Reply:
Not sure why I thought that __PX4_QURT_EAGLE applies to all  QURT targets, __PX4_QURT_EAGLE is probably enough. @dagar  Idea is that this problem might not affect boards with newer dsp image like those with APQ8096. I will try to test this, unfortunately adsp_info didn't work on my APQ8096 board.

Reply:
Since you have an `APQ8096` you would be the one able to answer this question :smile:, I don't have one.

Reply:
bsearch was missing on APQ8096 as well, didn't expect that. I'll remove the ifdef for now. Just in case I might later ask in Qualcom forum about this.

Reply:
Sounds good. I think the rest is good to go after the define is changed.

Reply:
APQ8096 test was done using hello world project instead of PX4 as I couldn't get PX4 to build for APQ8096.

Reply:
Removed the define and ifdef for now. It can be added back once someone finds a platform which isn't affected by this problem.